docs/application/vstools/Tizen/import.md

```diff
@@ -0,0 +1,21 @@
+# Importing Tizen Studio Projects Into Visual Studio
+
+The following sections explain how to use Visual Studio Extension for Tizen to import your projects created in Tizen Studio to Visual Studio.
+
+> [!NOTE]
+> Ensure that the Tizen Studio project was exported to CLI before importing to Visual Studio
```

Comment:
```suggestion
> Ensure that the Tizen Studio project was exported to CLI before importing to Visual Studio.
```

Reply:
@safir-srbd  I have made the suggested change. Thanks